README.md

```diff
@@ -1,46 +1,21 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+# Debate Deck
+
+## Development to do list
+
+### Needed for MVP
+- Understand how redux works with state management
+- Figure out how to manage users
+- Integrate firebase authentication for login and signup
```

Comment:
I think this project would be a lot more powerful if it didn't have to rely on firebase. This way we can deploy it anywhere and not have to pay firebase if this became commercial

Reply:
At the very least I think it might be worth it to use firebase auth because it can manage all the complicated stuff like guest accounts, email confirmations, password resetting, signing up with google/facebook/github etc. Also, I have like $400 from the school I can use, and I doubt it will cost me much for the dozen or so users I'll ever get lmao. 

But also, maybe I don't need all the fancy authentication things, and can just rely on a barebones username/password signup.

Reply:
I think the Google / Facebook auth is overkill for the scope of this project. The only benefit it would bring would be to prevent spam accounts or to pay to use Google / Facebook analytics.